mllib/src/main/scala/org/apache/spark/mllib/evaluation/BinaryClassificationMetrics.scala

```diff
@@ -26,7 +26,7 @@ import org.apache.spark.sql.DataFrame
 /**
  * Evaluator for binary classification.
  *
- * @param scoreAndLabels an RDD of (score, label) pairs.
+ * @param scoreAndLabelsWithOptWeight an RDD of (score, label, weight) tuples.
```

Comment:
It can also be a `(score, label)` tuple right?

Reply:
yes, good point, will update doc

Reply:
done!